From 889e1cc3dd628dc828fcf185f7c0a9f82baa441c Mon Sep 17 00:00:00 2001
From: Robert Kanter <rkanter@cloudera.com>
Date: Mon, 8 Oct 2012 09:37:15 -0700
Subject: [PATCH 146/795] MR1: CLOUDERA-BUILD. fix to allow hadoop-test jar to run in MR1 (CDH-8097)
 (cherry picked from commit 97d3600bc1c1772322079c8dd28d6c33e257dd68)

(cherry picked from commit 89efeb2ffadf7da768d85c36e12160c6142c98de)
---
 src/test/org/apache/hadoop/test/AllTestDriver.java |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/src/test/org/apache/hadoop/test/AllTestDriver.java b/src/test/org/apache/hadoop/test/AllTestDriver.java
index d78a3f3..b3287eb 100644
--- a/src/test/org/apache/hadoop/test/AllTestDriver.java
+++ b/src/test/org/apache/hadoop/test/AllTestDriver.java
@@ -37,7 +37,6 @@
 import org.apache.hadoop.io.TestArrayFile;
 import org.apache.hadoop.io.TestSequenceFile;
 import org.apache.hadoop.io.TestSetFile;
-import org.apache.hadoop.ipc.TestIPC;
 import org.apache.hadoop.ipc.TestRPC;
 import org.apache.hadoop.mapred.ThreadedMapBenchmark;
 
@@ -60,7 +59,6 @@ public static void main(String argv[]){
       pgd.addClass("testsetfile", TestSetFile.class, "A test for flat files of binary key/value pairs.");
       pgd.addClass("testarrayfile", TestArrayFile.class, "A test for flat files of binary key/value pairs.");
       pgd.addClass("testrpc", TestRPC.class, "A test for rpc.");
-      pgd.addClass("testipc", TestIPC.class, "A test for ipc.");
       pgd.addClass("testsequencefileinputformat", TestSequenceFileInputFormat.class, "A test for sequence file input format.");
       pgd.addClass("testtextinputformat", TestTextInputFormat.class, "A test for text input format.");
       pgd.addClass("TestDFSIO", TestDFSIO.class, "Distributed i/o benchmark.");
-- 
1.7.0.4

